src/patternfly/components/MenuToggle/examples/MenuToggle.md

```diff
@@ -269,6 +269,23 @@ import './MenuToggle.css'
 {{/menu-toggle}}
 ```
 
+### Small
+```hbs
+{{> menu-toggle menu-toggle--IsPlain=true menu-toggle--IsSmall=true menu-toggle--attribute='aria-label="Actions"' menu-toggle--icon="ellipsis-v"}}
+&nbsp;
+{{> menu-toggle menu-toggle--IsPlain=true menu-toggle--IsSmall=true menu-toggle--IsExpanded=true menu-toggle--attribute='aria-label="Actions"' menu-toggle--icon="ellipsis-v"}}
+&nbsp;
+{{> menu-toggle menu-toggle--IsPlain=true menu-toggle--IsSmall=true menu-toggle--IsDisabled="true" menu-toggle--attribute='aria-label="Actions"' menu-toggle--icon="ellipsis-v"}}
+```
+
+### Small with text
+```hbs
+{{> menu-toggle menu-toggle--IsPlain=true menu-toggle--text='Collapsed'}}
```

Comment:
These toggles also need the `menu-toggle--IsText=true` passed in to render the caret icon.